### ISSUE TITLE
Finetuning lr

### ISSUE THREAD

**Issue by WillQuCD** [NVIDIA/NeMo#733]
Great work!
Not a problem or an issue, just my experience to share with you guys.

When I fine tune my small dataset (10h) on the pretrained quartznet15x5 model, I found the performance highly depends on lr scales.

Following is the model performance on different lr scales, I get the best finetuning result with 0.00015. 
![image](https://user-images.githubusercontent.com/28244700/84748553-171fec00-afb9-11ea-8f5e-9e1fd8e054f1.png)

It is probably not the best lr and it is obviously time-consuming to try every possibility. So I am wondering if there's any strategy that can automatically choose the best lr?

**Comment by Slyne** [comment]
Is the y-axis training loss or wer?
How about the dev/test loss ?

**Comment by WillQuCD** [comment]
> Is the y-axis training loss or wer?
> How about the dev/test loss ?

The figure shows dev wer above. I didn't draw the test wer curve. It is consistent with the performance on dev.

**Comment by okuchaiev** [comment]
what we've observed is that when fine-tuning from pre-trained model, the learning rate should be ~10x compared to "training from scratch" case. See this paper for experiments: https://arxiv.org/abs/2005.04290

**Comment by maxschwendeler** [comment]
> what we've observed is that when fine-tuning from pre-trained model, the learning rate should be ~10x compared to "training from scratch" case. See this paper for experiments: https://arxiv.org/abs/2005.04290

What is your suggestions about the ratio of the fine-tuning dataset and the pre-training dataset to prevent the _catastrophic forgetting on the datasets used for pre-training_?

**Comment by csetanmayjain** [comment]
Hi @okuchaiev 

Thanks for the insights. I'm a bit doubtful about fine-tuning. Should I start the optim.lr with a value 10 times smaller than the base model, or with sched.min_lr? If it's sched.min_lr, then what should be the optimal value to start with for optim.lr? And if it's optim.lr, then what should be the optimal minimum value for sched.min_lr?
Can you please provide insights about this, or what optimal value did you choose for this while fine-tuning to get optimal results?
Thanks
